cmd/dagger/client_add.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"dagger.io/dagger"
+	"github.com/dagger/dagger/engine/client"
+	"github.com/dagger/dagger/engine/client/pathutil"
+	"github.com/spf13/cobra"
+)
+
+var (
+	generator string
+	localSDK bool
+)
+
+func init() {
+	clientAddCmd.Flags().StringVar(&generator, "generator", "", "Generator to use to generate the client")
+	clientAddCmd.Flags().BoolVar(&localSDK, "local-sdk", false, "Use local SDK dependency")
+}
+
+var clientAddCmd = &cobra.Command{
```

Comment:
Hm, so if I want to regenerate this, what do I do? Do I run the add command again?

It *feels* like we might want to just be able to run `dagger develop` and it would re-generate any existing code. Do I need to specify the generator again, and set the right directory again?

*Ideally* it feels like `dagger develop` would just work here :thinking: Or at least a `dagger client generate` or `dagger client sync`.

Don't really know, but the generated clients *should* end up in the `dagger.json` right so we can easily regen them? cc @shykes I know you had ideas around the CLI here.

Reply:
I agree with this entirely. I'm a novice when it comes to Dagger CLI but that does mean I can offer a novice's opinion:

When my module is set up, anything at all seems out of date or missing, I always hit `dagger develop`. 

If that doesn't work, I have to start digging.

Reply:
> Don't really know, but the generated clients should end up in the dagger.json right so we can easily regen them? cc @shykes I know you had ideas around the CLI here.

You may have multiple clients at some points, but I agree, having a `client` section in the daggerJSON make sense

```
{
  "client": [
     {
         "generator": "go"
         "outputPath": "./dagger" 
     }
  ]
}
```